executor/executor_test.go

```diff
@@ -989,6 +989,17 @@ func (s *testSuite) TestJSON(c *C) {
 	result = tk.MustQuery(`select a from test_json tj where a = "string"`)
 	result.Check(testkit.Rows(`"string"`))
 
+	// check some DDL limits for TEXT/BLOB/JSON column.
+	var err error
+	_, err = tk.Exec(`create table test_bad_json(a json default '{}')`)
+	c.Assert(err, NotNil)
```

Comment:
You should check the error code.
How can you make sure the error is not table already exists?